src/example-types/terms_stats.js

```diff
@@ -4,7 +4,9 @@ let { buildRegexQueryForWords } = require('../regex')
 let { getField } = require('../fields')
 
 let orderField = ({ include, order = 'sum' }) =>
-  include ? `${order}.value` : `stats.${order}`
+  include
+    ? `${_.replace('count', 'value_count', order)}.value`
```

Comment:
This should only be done if it's not the `stats` case right? Because if its stats, the field is actually just `count`

Reply:
More specifically if there's no `includes` (or if there is, that `stats`/`extended_stats` arent present. Perhaps a better check would be to do this only if `count` is explicitly present in includes

Reply:
If there is `include` and the `order` is `count` then it should always be replaced by `value_count`.
When will that not be the case?
Just like we always replace `count` with `value_count` in the `include` if it exists.

Reply:
There will alway be an exception if `order` is not a value in the include. For example they could say `order = 'sum'` and the `include  = ['count']` then elastic search will throw an exception about a none existing path but we are not checking for that because ES does. Are you saying we should check that the order should always be in the include, and if it does, what should we do? (